### ISSUE TITLE
Refining config/project.json

### ISSUE THREAD

**Issue by furzeface** [cartridge/cartridge-static-html#3]
# Project version:

**v1.0.0**
## Issue type:
- [x] Change proposal

in `_config/project.json` it's a bit ambiguous where the build artefacts go:

``` json
{
  "dirs": {
    "build": "build/_client",
    "dest": "public/_client"
  },
  "paths": {
    "dest": {
      "static_html": "build/",
      "sass": "public/_client/styles/",
      "scripts": "public/_client/scripts/",
      "images": "public/_client/images/"
    }
  }
}
```

(reduced case) ☝️ 

Some are in `build` and some `public`. @tawashley said that he manually changes them all to be the same each time he sets up a project with _Cartridge_. Should we decide on one standard for this?

:octocat:

**Comment by tawashley** [comment]
From what I understand, the `build` and `dest` keys were to allow flexibility (for when doing an express sites)?

To add onto what @furzeface mentioned, the two cartridge projects I've worked on, I have changed `dest` to be the same as `build` so that I know all of the output is in one place and not split.

TL;DR - Clarification on `build` and `dest` I think should be discussed and also whether the two being separate is at all required.

**Comment by furzeface** [comment]
Ace, when this is decided I will update issue #4 also. Thanks.

**Comment by bmds** [comment]
I don't think that they need to be split, i normally merge them

**Comment by furzeface** [comment]
Splendid, shall we have a consensus on which to use, `build` or `public`, in the meeting later today?

**Comment by furzeface** [comment]
public

**Comment by tawashley** [comment]
Work has been done on cartridge, to unify the build/dest project.json keys. With 1.2.0 of cartridge-static-html, similar has been done, with compiled views being added to the default directory of `public`. This issue can now be closed

**Comment by furzeface** [comment]
LGTM
